CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ### 0.11.2 (Next)
 
 * Your contribution here.
+* [#215](https://github.com/slack-ruby/slack-ruby-client/pull/215): Add more supports for cursor pagination - [@hotwatermorning](https://github.com/hotwatermorning).
```

Comment:
Please update this with all the changes in separate items by looking at the diff. For example "Added `apps_permissions_users_list`" or "`files_info`, `...` now support pagination". Thx.